src/services/user-service.js

```diff
@@ -117,7 +117,7 @@ class UserService {
       passwordHash: await this._hashPassword(password),
       email: lowerCasedEmail,
       roles,
-      expires: verified ? null : moment.utc().add(PENDING_USER_REGISTRATION_EXPIRATION_IN_HOURS, 'hours').toDate(),
+      expires: verified ? null : add(new Date(), PENDING_USER_REGISTRATION_EXPIRATION_IN_HOURS),
```

Comment:
Same here, it's not UTC

Reply:
I realized that we have a `date-time.js` in the utils folder with a function `now` that returns `new Date()`. It seems to be used only once in the document service for the `createdOn` date. This has always been wrongly using the server date instead of UTC, so maybe you could also change that, as you're on the subject anyways? (We're lucky that the Fargate server time is UTC, so all the DB times are correct, but we should make the code location-independent) :-)

Reply:
Hey, I will. The strange thing is that the new Date() was giving me UTC time in the console, or at least I thought it was.

Reply:
That's true, I hadn't noticed this change 🤦‍♀️ 
In the console you can actually see it's not UTC as the +1 timezone mentioned. If it were UTC the current time would be 08:12.